.github/workflows/__release-workflow.yaml

```diff
@@ -147,10 +147,18 @@ jobs:
         with:
           install: false
 
+      - name: Get Kong Enterprise License
+        if: secrets.op-service-account-token != ''
+        id: get-license
+        uses: Kong/kong-license@master
+        with:
+          # OP (1Password) token is used to fetch the Kong Enterprise License from 1Password.
+          op-token: ${{ secrets.op-service-account-token }}
+
       - name: integration tests
         run: make test.integration
         env:
-          KONG_LICENSE_DATA: ${{ secrets.kong-license-data }}
+          KONG_LICENSE_DATA: ${{ steps.get-license.outputs.license }} # The license is optional for OSS tests.
```

Comment:
Why this doesn't default to `''` as it does in line 208?

Reply:
Overlooked that, PTAL.